app/models/forms/application/savings_investment.rb

```diff
@@ -6,17 +6,18 @@ class SavingsInvestment < ::FormObject
 
       def self.permitted_attributes
         {
-          threshold_exceeded: Boolean,
-          partner_over_61: Boolean,
-          high_threshold_exceeded: Boolean
+          min_threshold_exceeded: Boolean,
+          over_61: Boolean,
+          max_threshold_exceeded: Boolean,
+          amount: Decimal
```

Comment:
In the public app I assumed we're not dealing with pence, only pounds, so I used `Integer`. I'll try to find out what's correct.